d3/query-graphs.js

```diff
@@ -592,6 +594,47 @@ function drawQueryTree(target, treeData) {
         update(d);
     }
 
+    // Dash tween to make the highlighted edges animate from start node to end node
+    var tweenDash = function() {
+        var l = this.getTotalLength();
+        var i = d3.interpolateString("0," + l, l + "," + l);
```

Comment:
This gives are very nice visual effect in case I hover the target (i.e the `ExplicitScan`) since it will start drawing a solid line from the `ExplicitScan` to its corresponding `GroupBy`.

However, if I hover the `GroupBy` instead of the `ExplicitScan` the line is drawn towards the hovered `GroupBy` node and this feels a bit unnatural to me. Especially given that when hovering the `GroupBy`, I am mentally focusing on that `GroupBy` node and the solid line only arrives there ~0.75 seconds after the hover event

Reply:
Will obviate difference by making hover highlight immediate, but then we lose directionality. Is direction, e.g., an ExplicitScan is a consumer, useful?

Reply:
Added flow directionality for crosslinks.

Reply:
looks really nice! :)